ConservationInternational/resilienceatlas
# Install transifex

Install transifex
---

Install transifex, add language selector, start adding translation components

- Transifex has been installed with a temporary account of Conservation international. 

- A new language selector has been added to the page on the right of the navbar.

- We have 6 fixed languages to translate.

- Currently only the string Journey from the navbar is translated to spanish to test.
- The rest of the strings marked as translatable should show differently on languages different than english (ȦȦɃǾǾŬŬŦ)  as we are using PseudoTranslationPolicy
- Only part of the UI is translated for now. Some missing pieces from the map and logged in pages have to be marked as translatable. This will be done on a different PR.

## Testing instructions

Change the language on the Language selector and check that you see the translation of journeys in spanish and the strings showing differently on the rest of strings. 

## Tracking

https://vizzuality.atlassian.net/browse/RA2-95?atlOrigin=eyJpIjoiZGNhYzk2YTE1Y2IzNGZlNTkzN2FkZjYyY2QzYzJlYmIiLCJwIjoiaiJ9
